sqlgen/mysql.go

```diff
@@ -21,7 +21,12 @@ func (w *SimpleWhere) ToSQL() (string, []interface{}) {
 				buffer.WriteString(" AND ")
 			}
 			buffer.WriteString(column)
-			buffer.WriteString(" = ?")
+			val := w.Values[i]
+			if val == nil {
```

Comment:
maybe just `if w.Values[i] == nil` to save a temporary variable.

Reply:
updated